aboutpage.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/56d99cc5b2.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <link rel="stylesheet" href="./aboutpage.css"/>
+    <title>Capstone project</title>
+  </head>
+  <body>
+    <header class="head">
+      <div class="homepage">
+        <div class="nav-list-one">
+          <ul class="list">
+            <li>
+              <a class="header-icon" href="#"
+                ><i class="fa-brands fa-facebook-f"></i
+              ></a>
+            </li>
+            <li>
+              <a class="header-icon" href="#"
+                ><i class="fa-brands fa-twitter"></i
+              ></a>
+            </li>
+            <li>English</li>
+            <li>My page</li>
+            <li>Log out</li>
+          </ul>
+        </div>
+
+        <div class="all">
+           <nav class="nav-bar">
+            <a class="nav-img" href="./images/RO.jpeg"><img id="nav-img" src="./images/RO.jpeg" alt=""></a>
+            <ul class="navigation">
+              <li class="nav-item"><a href="./index.html" class="nav-link">Home</a></li>
```

Comment:
- [ ] Please always remember to include a descriptive meaning to the image using the `alt`. i made a detailed review on the summary, please check it out 👏